config/sync/core.base_field_override.node.state_directory_record.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: fec2315e-00a5-475b-98a7-f112199f2abe
```

Comment:
I think all these yml files are (or should be) in the pull request for USAGOV-30, right?